odoo/addons/base/models/ir_actions.py

```diff
@@ -65,6 +68,19 @@ def unlink(self):
         self.clear_caches()
         return res
 
+    def _ensure_group_xml_id(self):
+        """Create an external identifier to the groups used in the actions
+        This is required for the cache of get_bindings"""
+        if 'groups_id' in self._fields:
+            for group in self.groups_id:
+                if not group.get_external_id()[group.id]:
+                    self.env['ir.model.data'].create({
```

Comment:
Can be nice to batch it (create + get_external_id) ? For now, each view are generally create/write one by one, but it might not the case tomorrow (we plan to batch creation of records coming from xml data).